c10/core/alignment.h

```diff
@@ -20,11 +20,6 @@ constexpr size_t gPagesize = 4096;
 // for buffers of size 2MB or larger to avoid memory bloating
 constexpr size_t gAlloc_threshold_thp = static_cast<size_t>(2) * 1024 * 1024;
 
-// Cache line size used to avoid false sharing between threads. Falls back to 64
-// bytes if C++17 feature is unavailable.
-#ifdef __cpp_lib_hardware_interference_size
-using std::hardware_destructive_interference_size;
-#else
+// Cache line size used to avoid false sharing between threads.
 constexpr std::size_t hardware_destructive_interference_size = 64;
```

Comment:
Actually, should we use define c10::hardware_destructive_interference_size = std::hardware_destructive_interference_size; if __cpp_lib_hardware_interference_size is defined?

Reply:
This will introduce the same warning

```bash
vllm-user-6/pytorch/c10/core/alignment.h:26:10: warning: use of ‘std::hardware_destructive_interference_size’ [-Winterference-size]
```